onnxruntime/core/framework/thread_safe_map.h

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <mutex>
+#include <unordered_map>
+
+// Thread-safe container wrapper for avoiding potential racing conditions
+// when using parallel executor.
```

Comment:
>parallel executor [](start = 14, length = 17)

no one is using it AFAIK..

Reply:
Maybe not many users. But this ORT release was blocked because of that.

---
In reply to: [462617744](https://github.com/microsoft/onnxruntime/pull/4547#discussion_r462617744) [](ancestors = 462617744)